src/components/common/TmSessionHardware.vue

```diff
@@ -135,7 +135,7 @@ export default {
       this.status = `detect`
       this.address = null
       try {
-        this.address = await getAddressFromLedger(this.networkId)
+        this.address = await getAddressFromLedger(this.networkId, this.$apollo)
```

Comment:
why do we have to pass `$apollo` as an argument?

Reply:
it needs it to query for the address prefix in `ledger.js`, lines 48-59. 
Before we were using the hardcoded dictionary and now we are using the API.